sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateClientBuilder.java

```diff
@@ -357,6 +399,7 @@ public CertificateClientBuilder pipeline(HttpPipeline pipeline) {
      *
      * @return The updated {@link CertificateClientBuilder} object.
      */
+    @Override
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores some values in the Configuration object. Should we add a note like you did in other methods on this same class?